src/actions/compileManuscript.ts

```
@@ -0,0 +1,128 @@
+
+type Metadata = {
+    dpi: BoundingBox;
+    horizontal_lines?: BoundingBox[];
+    rectangles?: BoundingBox[];
+};
+
+type SuryaPage = SuryaPageOcrResult & {
+    page: number;
+};
+
+const promptJsonFile = async (message: string) => {
```

ESLint

[error] 'promptJsonFile' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)